### ISSUE TITLE
/usr/lib64/libldap.so.2: undefined reference to EVP_md2@OPENSSL_3.0.0

### ISSUE THREAD

**Issue by umagmrit** [openssl/openssl#23638]
Getting below issue

OS : Rocky Linux 9

`/usr/lib64/libldap.so.2: undefined reference to EVP_md2@OPENSSL_3.0.0
`

```
$ strings /usr/lib64/libldap.so.2 | grep EVP_md2
EVP_md2
```

```
$ rpm -qf /usr/lib64/libldap.so.2
openldap-2.6.2-3.el9.x86_64
```

I have compiled `openssl 3.0.9` from source without `enable-md2` option . Do I need to use `enable-md2` to fix this issue ? is there any security concerns with `enable-md2` option ?

**Comment by t8m** [comment]
Yes, you need to add enable-md2 to fix this issue. Unless MD2 is actually used, there should be no issue.
